src/index.html

```diff
@@ -11,7 +11,184 @@
+            required
+            >
+          </label>
+        </fieldset>
+      </div>
+
+      <div class="group">
+        <fieldset>
+          <legend>Registration</legend>
+          <label class="label-inline label-margin" for="email">
+            E-mail:
+            <input 
+            type="email" 
+            id="email" 
+            name="email" 
+            placeholder="email@example.com"
+            >
+          </label>
+          <label>Password:
+            <input 
+            type="password" 
+            id="pwd" 
+            name="pwd" 
+            required 
+            minlength="8" 
+            maxlength="16"
+            >
+          </label>
+        </fieldset>
+      </div>
+    
+      <div class="group">
+        <fieldset>
+          <legend>An interesting fact about you!</legend>
+          <label class="label-margin">
+            Do you love cats?
+            <label>
+              <input 
+              type="radio" 
+              id="cats-yes" 
+              name="cats" 
+              value="yes" 
+              required
+              >
+              Yes
+            </label>
+          
+            <label>
+              <input 
+              type="radio" 
+              id="cats-no" 
+              name="cats" 
+              value="no" 
+              required
+              >
+            No
+            </label>
+          </label>
+          
+          <label for="color" class="label-margin">
+            What is your favorite color?
+            <input 
+            type="color" 
+            id="color" 
+            name="color" 
+            required
+            >
+          </label>
+          <label for="bedTime" class="label-margin">
+            What time do you go to bed?
+            <input 
+                type="time" 
+                id="bedTime" 
+                name="bedTime" 
+                required
+                step="1"
+            >
+          </label>
+          <label class="label-margin">
+            What are your favorite brand of cars?
+            <select name="car" multiple>
+              <option value="bmw">BMW</option>
+              <option value="audi">Audi</option>
+              <option value="Lada">Lada</option>
+            </select>
+          </label>
+          <label>
+            How do you rate our work?
+            <input 
+            type="range" 
+            min="0" 
+            max="10" 
+            step="1" 
+            name="rating" 
+            id="rating"
+            value="0"
+            >
+          </label>
+        </fieldset>
+      </div>
+  
+      <div class="group">
+        <fieldset>
+          <legend>Additional info:</legend>
+          <label>
+            Comments:
+            <textarea 
+              id="comments" 
+              name="comments" 
+              rows="2" 
+              cols="20"
+            >
+            </textarea>
+          </label>
+          <br>
```

Comment:
add class and remove `<br>`
```suggestion
          <label class="label-margin">
            Comments:
            <textarea 
              id="comments" 
              name="comments" 
              rows="2" 
              cols="20"
            >
            </textarea>
          </label>
```